commands/cli.go

```diff
@@ -135,3 +142,107 @@ func awsNuke(c *cli.Context) error {
 
 	return nil
 }
+
+func awsDefaults(c *cli.Context) error {
+	logging.Logger.Infoln("Identifying enabled regions")
+	regions, err := aws.GetEnabledRegions()
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+	for _, region := range regions {
+		logging.Logger.Infof("Found enabled region %s", region)
+	}
+
+	err = defaultVpcs(c, regions)
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+
+	err = defaultSecurityGroups(c, regions)
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+	return nil
+}
+
+func defaultVpcs(c *cli.Context, regions []string) error {
+	logging.Logger.Infof("Discovering default VPCs")
+	vpcPerRegion := aws.NewVpcPerRegion(regions)
+	vpcPerRegion, err := aws.GetDefaultVpcs(vpcPerRegion)
```

Comment:
Out of curiosity, why can't we pass in the regions and lookup the defaults in each one there?

Reply:
See comment above re: the service client. This is different between the default VPCs and the security group because with default VPCs, there is exactly one per region. With default security groups, there is one per VPC, and potentially many VPCs per region. That's why it's handled differently between the two.

Reply:
Thanks for the explanation!